docker-compose.test.yml

```diff
@@ -0,0 +1,59 @@
+version: "2.1"
+services:
+  postgres:
+    image: postgres:14-alpine
+    restart: always
+    environment:
+      POSTGRES_USER: docker
+      POSTGRES_PASSWORD: docker
+      POSTGRES_DB: postgres
+    ports:
+      - "5432"
+      - "5431:5432"
+    networks:
+      - internal
+  redis:
+    image: redis:alpine
+    restart: always
+    ports:
+      - "6379"
+      - "6378:6379"
+    networks:
+      - internal
+  loki: 
+    image: grafana/loki:2.0.0
+    restart: always
+    ports:
+      - "9095"
+      - "3100"
+      - "9094:9095"
+      - "3101:3100"
+    networks:
+      - internal
+  sut:
+    image: balena/pinejs-sut:latest
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: sut
+    depends_on:
+      - postgres
+      - redis
+      - loki
+    environment:
+      LOGLEVEL: error
+      API_HOST: 127.0.0.1 
+      DATABASE_URL: postgres://docker:docker@postgres:5432/postgres 
+      DEBUG: 
+      PORT: 80 
+      REDIS_HOST: "redis:6379" 
+      LOKI_HOST: loki
+      LOKI_PORT: 3100
+      LOKI_WRITE_PCT: 100 
```

Comment:
These don't exist in pinejs itself

Reply:
copy paste left form open-balena-api. deleted